brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
We test that using the `-x` option currently and anyway it is valid for it not to exist even if no QEMU is specified: native builds...